sdk/cosmos/azure-cosmos/azure/cosmos/aio/_retry_utility_async.py

```diff
@@ -189,6 +193,15 @@ async def ExecuteAsync(client, global_endpoint_manager, function, *args, **kwarg
                 if kwargs['timeout'] <= 0:
                     raise exceptions.CosmosClientTimeoutError()
 
+        except ServiceRequestError:
+            _handle_service_request_retries(request, client, service_response_retry_policy, args)
+
+        except ServiceResponseError as e:
+            if e.exc_type in [ConnectionTimeoutError, ServerTimeoutError]:
```

Comment:
We are tracking different types here when compared to `_retry_utility.py` - is this expected?

Reply:
yep, some are based on the async transport and the others on the sync transport